extensions/machine-learning/src/views/models/localModelsComponent.ts

```diff
@@ -38,8 +38,11 @@ export class LocalModelsComponent extends ModelViewBase implements IDataComponen
 			width: this.componentMaxLength - this.browseButtonMaxLength - this.spaceBetweenComponentsLength
 		}).component();
 		this._localBrowse = modelBuilder.button().withProperties({
-			label: constants.browseModels,
-			width: this.browseButtonMaxLength
+			iconPath: this.asAbsolutePath('images/browseLocal.svg'),
```

Comment:
please make sure this looks ok in dark and light theme both

Reply:
It does not look good in dark / high contrast. I'll code it for both a light and dark path for now.